frontend/src/components/positions-list.js

```diff
@@ -39,8 +52,81 @@ const DEFAULT_COLUMNS = [
         Header: "Contract Template",
         accessor: "contract_template.template_name",
     },
+    {
+        Header: "Duties",
+        accessor: "duties",
+    },
+    {
+        Header: "Instructor Preferences",
```

Comment:
Instructor preferences are for TAs, not for positions. This doesn't belong here...

Reply:
I had just added all the columns from the api doc for GET positions. If there are any else that shouldn't be viewable, please let me know so I can add them to the blacklst